packages/react-components/react-link/library/package.json

```diff
@@ -24,6 +24,7 @@
     "@fluentui/react-shared-contexts": "^9.21.2",
     "@fluentui/react-tabster": "^9.23.3",
     "@fluentui/react-theme": "^9.1.24",
+    "@fluentui/semantic-tokens": "^0.0.0-alpha.1",
```

Comment:
Should we be importing this via react-theme for Fluentui component usage?

Reply:
like re-export through the `react-theme` package? I think the less we tangle dependencies the better IMHO.